src/analyses/threadId.ml

```diff
@@ -104,13 +104,19 @@ struct
     else
       None
 
+  let node_for_ctx ctx = match (ctx.ask Queries.ThreadId) with
+    | `Lifted node -> node
+    | _ -> ctx.prev_node
+
   let threadenter ctx lval f args =
-    let+ tid = create_tid ctx.local ctx.prev_node f in
+    (* x *)
```

Comment:
why the `(* x *)`?

Reply:
The PR is still in a draft state, needs tests and a cleanup still!